e2e/bin/nodes-opts

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+nodes=$(terraform output -json -state=terraform/terraform.tfstate | jq -r '(.linux_clients,.servers).value[]')
```

Comment:
As long as we're in here updating these scripts to make them position independent, maybe we can fix this state path so that you don't have to run this from the root of `e2e/`?